src/scripts/app.ts

```diff
@@ -1001,15 +1001,9 @@ export class ComfyApp {
    */
   async setup(canvasEl: HTMLCanvasElement) {
     this.canvasEl = canvasEl
-    // Show menu container for GraphView.
```

Comment:
No longer necessary as the setting value initialization will trigger the vue watcher to properly set the display attr of legacy menu.